config/default/kustomization.yaml

```diff
@@ -19,13 +19,12 @@ namePrefix: aibrix-
 
 resources:
 - ../namespace
-- ../crd
 - ../rbac
 - ../manager
 - ../gateway
 - ../metadata
 - ../gpu-optimizer
-- ../dependency/kuberay-operator
+- ../kuberay-operator
```

Comment:
why do we need to change kuberay-operator dir? That can still be part of dependency along with envoy-gateway.